integration_test/tests/common/mod.rs

```diff
@@ -324,17 +357,25 @@ pub fn fetch_and_get_data(
 }
 
 #[allow(dead_code)]
-///  Helper function for checking resultset metadata
+/// Helper function for checking result set metadata. It optionally takes a list
+/// of attributes to get. If none is provided, the following defaults will be
+/// checked. It also optionally invokes SQLDescribeColW.
```

Comment:
nit: might be bikeshedding and feel free to do whatever you think seems clearest; but could be clearer if this comment points out that this function should get all column metadata, and some tools do this by calling SQLColAttribute for every attribute, and some get the bulk of attributes through SQLDescribeCol then request addtional attributes ad-hoc